Core/StartupArgManager.cs

```diff
@@ -37,5 +36,27 @@ public static string[] MergeStartupArgs(IEnumerable<string> cmdArgs)
                 return startupArgs.ToArray();
             }
         }
+
+        public static void MigrateArgsFile()
+        {
+            try
+            {
+                string OldFileLocation = "laargs.txt";
+
+                if (!File.Exists(StartupArgsPath)) { return; }
```

Comment:
1. Return should be in a new line, brackets shouldn't be used.
2. That if completely breaks the migration. User launches the new LA, new file doesn't exist, so the migration is not performed. It will be never performed unless the user creates a file in the new location. And if there is already a new config then we shouldn't touch it, because the user already put something there. The only thing we should do in such case is deleting the old config if it's an empty file.

Reply:
Hopefully the new stuff works.